src/dodal/devices/diamond_filter.py

```diff
@@ -25,22 +25,10 @@ class I04Filters(_Filters):
 T = TypeVar("T", bound=_Filters)
 
 
-class DiamondFilter(StandardReadable, Generic[T]):
+class DiamondFilter(Positioner1D[T]):
     """
     A filter set that is used to reduce the heat load on the monochromator.
 
     It has 4 slots that can contain filters of different thickness. Changing the thickness
     signal will move the filter set to select this filter.
     """
-
-    def __init__(
-        self,
-        prefix: str,
-        data_type: type[T],
-        name: str = "",
-    ) -> None:
-        with self.add_children_as_readables():
-            self.y_motor = Motor(prefix + "Y")
```

Comment:
Is the y_motor position no longer required on i03/i04?
Must: This should at least be called out in the ticket body so it's clear that it's an intentional change if it is.